### ISSUE TITLE
Enable main branch as master branch too

### ISSUE THREAD

**Issue by laresbernardo** [r-lib/usethis#1510]
Given that `master` branch are slowly being deprecated and transformed to `main` branch, I propose enabling `main` as a valid master branch too.
```
> usethis::use_github()
✓ Checking that current branch is default branch ('master')
Error: Must be on branch 'master', not 'main'.
```

**Comment by malcolmbarrett** [comment]
usethis had had a lot of recent improvements to help this transition. See https://usethis.r-lib.org/reference/git-default-branch.html, for instance, in the development version. In general, usethis also handles the GitHub defaults well, too

**Comment by jennybc** [comment]
Yeah what I see from OP is from an old version of usethis. Even the current CRAN release will not show this. And yes the coming release has even more changes.

**Comment by laresbernardo** [comment]
Thanks! Tried with dev version and made sense all changes and worked OK
